psycopg_c/pyproject.toml

```
@@ -5,8 +5,8 @@ requires = [
     #   `[tool.setuptools.ext-modules]` in `pyproject.toml` is still
     #   *experimental* and likely to change in future releases
     #
-    "setuptools == 75.6.0; python_version >= '3.9'",
-    "setuptools == 75.3.0; python_version < '3.9'",  # last supported version
+    "setuptools == 75.6.0; python_version >= '3.10'",
+    "setuptools == 75.3.0; python_version < '3.10'",  # last supported version
```

I am not sure these are right.

---

Should I leave it unchanged?

---

I looked back into it. Probably we just drop the version, as 75.6 supports all the ones we need.

```suggestion
    "setuptools == 75.6.0"
```

Unpinning this version, as per note above, is unrelated to this MR, we will look into it later.